src/sparse/KokkosSparse_spmv.hpp

```diff
@@ -157,12 +157,23 @@ spmv (KokkosKernels::Experimental::Controls controls,
 
 #ifdef KOKKOSKERNELS_ENABLE_TPL_CUSPARSE
   //cuSPARSE does not support the conjugate mode (C), and cuSPARSE 9 only supports the normal (N) mode.
+  if(std::is_same<typename AMatrix_Internal::memory_space, Kokkos::CudaSpace>::value ||
+      std::is_same<typename AMatrix_Internal::memory_space, Kokkos::CudaUVMSpace>::value)
+  {
 #if (9000 <= CUDA_VERSION)
-  useFallback = useFallback || (mode[0] != NoTranspose[0]);
+    useFallback = useFallback || (mode[0] != NoTranspose[0]);
 #endif
 #if defined(CUSPARSE_VERSION) && (10300 <= CUSPARSE_VERSION)
-  useFallback = useFallback || (mode[0] == Conjugate[0]);
+    useFallback = useFallback || (mode[0] == Conjugate[0]);
```

Comment:
Do we need to worry about conjugate transpose as well?

Reply:
@lucbv No, I made sure that both the old and new interfaces support modes N, T and H. (what we call H, mkl 17 calls C).